libsolidity/interface/CompilerStack.cpp

```diff
@@ -653,6 +664,12 @@ void CompilerStack::compileContract(
 		// CBOR-encoding of {"bzzr0": dev::swarmHash(metadata)}
 		bytes{0xa1, 0x65, 'b', 'z', 'z', 'r', '0', 0x58, 0x20} +
 		dev::swarmHash(metadata).asBytes();
+	if (!safeExperimentalFeature(_contract.sourceUnit().annotation().experimentalFeatures))
+		cborEncodedMetadata =
+		// CBOR-encoding of {"bzzr0": dev::swarmHash(metadata), "experimental": true}
+		bytes{0xa2, 0x65, 'b', 'z', 'z', 'r', '0', 0x58, 0x20} +
```

Comment:
yes, so basically everything apart from the first byte. CBOR is defined in a way so that it can be composed at byte-level, isn't it?